app/resources/numismatic_finds/numismatic_find.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+class NumismaticFind < Resource
+  include Valkyrie::Resource::AccessControls
+  attribute :member_ids, Valkyrie::Types::Array
```

Comment:
I don't think we need `member_ids`, because the Coins should link to the Finds instead of the other way around.  As with NumismaticAccessions, we could model the Coins of members, but we generally avoid having an object be a member of more than one parent if we can avoid it.

Reply:
Do you mean NumismaticCitations instead of NumismaticAccessions? The NumismaticAccessions have `member_ids`

Reply:
Oh, NumismaticAccessions shouldn't have `member_ids` — I'll make a ticket to remove them.

Reply:
oh ok. thanks.